tox.ini

```diff
@@ -33,7 +40,8 @@ commands =
     rm -f docs/modules.rst
     make -C docs clean
     make -C docs html
-    python setup.py check --restructuredtext --strict
+    python setup.py sdist bdist_wheel
+    twine check dist/*
```

Comment:
`twine` wasn't added to `requirements/doc.in`, and it looks like we don't run this code in Travis yet.  Probably should to avoid problems with package uploads on README verification.